docs/sessions/2026-04/2026-04-21-prancy-pudding-handoff.md

```
@@ -0,0 +1,232 @@
+
+`build` (Storybook webpack/TS loader) e `Lighthouse Performance Audit` continuam falhando em PRs frontend. **Não são required checks** (branch protection exige apenas `Backend Tests` + `Frontend Tests`). Stories documentadas:
+- `DEBT-CI-storybook-ts-loader`: diagnosticar loader config (2-4h)
+- `DEBT-CI-lighthouse-timeout`: fix tool hang OU mover para schedule main-only (2-3h)
+
+---
+
+## 2. Cascata de merge executada
+
+Ordem de merge estratégica (por dependência):
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)